ironfish/src/fileStores/config.ts

```diff
@@ -303,7 +303,7 @@ export class Config extends KeyStore<ConfigOptions> {
       tlsKeyPath: files.resolve(files.join(dataDir, 'certs', 'node-key.pem')),
       tlsCertPath: files.resolve(files.join(dataDir, 'certs', 'node-cert.pem')),
       maxPeers: 50,
-      minimumBlockConfirmations: 12,
+      minimumBlockConfirmations: 2,
```

Comment:
Is this an intentional change or for testing?

Reply:
Intentional, but I snuck it into this PR. I should have wrote it in the description.